ppml/scripts/generate-ssl.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Can we use this script for key generation?
https://github.com/intel-analytics/BigDL/blob/main/ppml/scripts/generate-keys.sh